src/assets/js/timezone-utils.js

```diff
@@ -0,0 +1,39 @@
+export const getHoursDiff = (date1, date2) =>
```

Comment:
any reason why not to use a function here?

Reply:
What do you mean by this? This is a function...